internal/core/src/exec/expression/UnaryExpr.cpp

```diff
@@ -552,12 +583,15 @@ PhyUnaryRangeFilterExpr::ExecArrayEqualForIndex(bool reverse) {
 
 template <typename ExprValueType>
 VectorPtr
-PhyUnaryRangeFilterExpr::ExecRangeVisitorImplJson(OffsetVector* input) {
+PhyUnaryRangeFilterExpr::ExecRangeVisitorImplJson(EvalCtx& context) {
     using GetType =
         std::conditional_t<std::is_same_v<ExprValueType, std::string>,
                            std::string_view,
                            ExprValueType>;
 
+    auto* input = context.get_offset_input();
+    const auto& bitmap_input = context.get_bitmap_input();
+    bool has_bitmap_input = !bitmap_input.empty();
```

Comment:
the. has_bitmap_input seems to be redundant. 
use bitmap_input.empty() seems to be good enough